package.json

```diff
@@ -40,5 +40,9 @@
     "truffle": "3.1.2",
     "truffle-contract": "^1.1.10",
     "truffle-hdwallet-provider": "0.0.3"
+  },
+  "dependencies": {
+    "eth-lightwallet": "^2.5.6",
+    "left-pad": "^1.1.3"
```

Comment:
In fact I just tried it with Node v8 and it actually fails if we don't put it in dev-dependencies.